teuthology/misc.py

```diff
@@ -1154,26 +1154,16 @@ def get_distro(ctx):
     """
     Get the name of the distro that we are using (usually the os_type).
     """
-    os_type = None
+    # ubuntu is our default distro choice
+    os_type = "ubuntu"
     if ctx.os_type:
         return ctx.os_type
 
     try:
-        os_type = ctx.config.get('os_type', None)
+        os_type = ctx.config.get('os_type', os_type)
```

Comment:
Do we need to worry about the possibility of ctx.config['os_type'] = None?

If for whatever reason that happens this function will return None and provisioning will fail.

Maybe default os_type=None at the top and then return os_type or "ubuntu" would be safer?